api/main.py

```diff
@@ -1,31 +1,113 @@
+import ujson
 from aiohttp import web
+from aiohttp_session import setup, get_session
 from loguru import logger
-from shared.project_settings import settings
-from database.utilities import apply_migrations
+
+from api.utilities import (
+    register_user,
+    login_user,
+    get_random_fact,
+    check_if_user_logged_in,
+    create_redis_storage,
+    get_random_rated_fact_info,
+    process_rating,
+    check_for_required_info_for_login,
+    check_for_required_info_for_registration,
+    check_for_required_info_to_rate_title
+)
 from database.models import connect_to_db, db
+from database.utilities import apply_migrations
+from shared.exceptions import PasswordError, LoginError
+from shared.project_settings import settings
 
 
+DATA_CHECK = {
+    'login': check_for_required_info_for_login,
+    'registration': check_for_required_info_for_registration,  # Check for the required keys in data to process request
+    'rate_process': check_for_required_info_to_rate_title,
+    }
 app_route = web.RouteTableDef()
 
 
-@app_route.get('/')
-async def index(request: web.Request):
-    return web.Response(text='test')
+@app_route.post('/registration')
+async def register(request: web.Request):
+    data = await request.json(loads=ujson.loads)
+    DATA_CHECK['registration'](data)
```

Comment:
Есть какая-то причина не написать
```suggestion
    check_for_required_info_for_registration(data)
```
?

Зачем нужно промежуточное звено в виде DATA_CHECK?